packages/api/test/e2e/cennzx.e2e.ts

```diff
@@ -138,7 +139,7 @@ describe('CENNZX RPC calls testing', () => {
         console.log('Amount of core in PLUG pool:', poolCoreAssetBalance.toString());
         // How much CENTRAPAY will it cost to buy 100 (amount) PLUG
         await expect(api.rpc.cennzx.buyPrice(CENTRAPAY, amount, PLUG)).rejects.toThrow(
-          'buyPrice(AssetToBuy: AssetId, Amount: Balance, AssetToPay: AssetId): Balance:: 2: Cannot exchange for requested amount.:'
+          '2: Cannot exchange for requested amount.:'
```

Comment:
Are the number and ":" meaningful for this message?

Reply:
This comes from node.. I am just displaying what comes from node's rpc call..